packages/react-magma-dom/src/components/Dropdown/DropdownExpandableMenuGroup.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
+import styled from '../../theme/styled';
+import { useGenerateId } from '../../utils';
+import { Accordion, AccordionProps } from '../Accordion';
+import { DropdownContext } from './Dropdown';
+
+/**
+ * @children required
+ */
+export interface DropdownExpandableMenuGroupProps
+  extends React.HTMLAttributes<HTMLDivElement> {
+  isInverse?: boolean;
+  testId?: string;
+}
+
+const StyledAccordion = styled(Accordion)`
+  border: none;
+`;
+
+export const DropdownExpandableMenuGroup = React.forwardRef<
+  HTMLDivElement,
+  AccordionProps
+>(props => {
+  const { children, id: defaultId, testId, ...other } = props;
+
+  const id = useGenerateId(defaultId);
+
+  const context = React.useContext(DropdownContext);
+
+  const countEachGroup = document.querySelectorAll('[role="group"]').length;
```

Comment:
Changed to `[role="expandable group"]`

Reply:
What if there's an accordion on the page? or just something totally unrelated that happens to have that same role. Is there a more specific way we can find this instead of query selecting?

Reply:
Pretty sure I figured out a way, and moved the check to `DropdownExpandablePanel`!